DEVELOPMENT.md

```diff
@@ -1,9 +1,17 @@
 # Contributing
-## Development tips
+
+## Testing
+
+First, prepare a database for the test.
+
+```sh
+bundle install
+RAILS_ENV=test bundle exec rake db:create db:migrate
```

Comment:
yes, but I don't like using `rails` instead of `rake`. And the documentation is also read by rails 4 user. So, I think `rake` is good in the documentation.